store/src/main/java/com/nytimes/android/external/store/base/experimental/Store.java

```diff
@@ -0,0 +1,64 @@
+package com.nytimes.android.external.store.base.experimental;
+
+
+import com.nytimes.android.external.store.base.impl.BarCode;
+
+import org.jetbrains.annotations.NotNull;
+
+import rx.Observable;
+
+/**
+ * a {@link com.nytimes.android.external.store.base.impl.StoreBuilder StoreBuilder}
+ * will return an instance of a store
+ * <p>
+ * A {@link Store  Store} can
+ * {@link Store#get(BarCode) Store.get() } cached data or
+ * force a call to {@link Store#fetch(BarCode) Store.fetch() }
+ * (skipping cache)
+ */
+public interface Store<T, V> {
```

Comment:
If you're going to introduce breaking changes anyway, just do it all in one go. Reduce tech debt.